src/api/link_delete.js

```diff
@@ -1,7 +1,10 @@
+const { createLogger } = require("../logger")
+
 class LinkDeleteHandler {
   constructor(uPortMgr, linkMgr) {
     this.uPortMgr = uPortMgr
     this.linkMgr = linkMgr
+    this.logger = createLogger({ name: "link_delete" })
```

Comment:
`api.link_delete` might make this easier to find in the code, by namespacing the logger names according to the module path.

same applies for others